lib/core.js

```diff
@@ -35,7 +35,8 @@ module.exports = {
 
     classData.componentWillUnmount = function() {
       var chart = this.state.chart;
-      chart.destroy();
+      if(chart)
```

Comment:
sorry for the nitpick, but could you wrap this if statement with braces?

Reply:
@mloisotto Do you have an estimation on when you can get to this? I am currently being blocked by it. If you can't let me know and I can take over.